db/migrate/20210830165844_create_users_table.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsersTable < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users_tables do |t|
+      t.string :email, null: false, unique: true
+      t.string :password, null: false
+      t.string :name
```

Comment:
could you please add timestamps here?

(weird cuz should have come automatically with rails command)